### ISSUE TITLE
 xml api's from tvdb

### ISSUE THREAD

**Issue by Blurayx** [WayneD/script.tv.show.next.aired#9]
Hi WayneD,
Do you plan on updating the old deprecated xml api's from tvdb
without going into too much detail please read the link from the kodi forum
Post: #3128
http://forum.kodi.tv/showthread.php?tid=235676&pid=2516808#pid2516808

Thanks

**Comment by WayneD** [comment]
This looks promising to make use of: https://github.com/marcelveldt/script.module.thetvdb

I'm not currently running Kodi, so if someone wants to work on changing the code to use that, please feel free. From a quick glance it looks like that code could handle all the updating and caching that Next-Aired currently does on its own.

**Comment by marcelveldt** [comment]
@WayneD yes I created the module with the thought in mind that it could also be used for the nextaired addon. If you are okay with it I can go ahead with adjusting your code to use the new tvdb module.

**Comment by marcelveldt** [comment]
I will implement the change to use the tvdb module. Do you want me to PR the changes to you so you can submit the changes to the Kodi repo yourself ?

**Comment by WayneD** [comment]
I'd be glad to submit the changes, if you'd like me to. However, in the long run it would be good for me to hand off the maintainer role to someone else, since I'm not using this software anymore. Do you want to take it over?

**Comment by marcelveldt** [comment]
Sorry for the late reply. I finished the refactor of the script. In the end it was almost a full rewrite :-)

https://github.com/marcelveldt/script.tv.show.next.aired

I'll be happy to take over the maintenance if you'd like. Just tell me what has your preference.

**Comment by WayneD** [comment]
> I'll be happy to take over the maintenance if you'd like.

Sure, that would be great. Why don't you go ahead and post something to the xbmc-addons mailing list (or whatever it's named these days) with your repo's URL and the branch to use (it's also good to add a tag with the release number). I don't remember what the steps were to take over as maintainer back when I first started helping out, so you might include a link to this thread if you think that will help (and have them feel free to contact me if they have any questions about the hand-off).
